apInitPassthru.py

```diff
@@ -296,6 +313,9 @@ def mlrs_open_passthrough(comport, baudrate, serialx, options=''):
         link.close()
         link = ardupilot_connect(apport, receiver_baud)
         time.sleep(0.5) # can't hurt
+    if 'scripting' in options:
+        ardupilot_set_scripting(link, serialx)
+        link = ardupilot_connect(apport, receiver_baud)
```

Comment:
no link.close() before ?

Reply:
The close must be done in ardupilot_set_scripting() because it needs to be done before the user is instructed to replug the USB cable while holding the rx boot button.  If it is still open when the USB is plugged in again, a different serial device will be assigned and we would have to find the new one.